src/components/FeedSingleTweet.tsx

```diff
@@ -118,7 +119,12 @@ const FeedSingleTweet = ({
               <div className="mx-1 text-secondary d-flex justify-content-center align-items-center">
                 ·
               </div>
-              <UnderlineHover className="text-secondary">
+              <UnderlineHover
```

Comment:
Did I understand correctly that when clicking on the date of creation, it redirects to a page with a specific tweet (by id)? Then why only when clicking on the date and not the entire tweet (the entire tweet field)